docker/r/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # define build arguments
-ARG RENKU_BASE=renku/renkulab-py:latest
+ARG RENKU_BASE=renku/renkulab-py:3.8-0.7.6
```

Comment:
btw, `latest` is there as a placeholder so we don't have to keep changing it. When the CI runs, it replaces this to use the base python image corresponding to the same commit.

Reply:
ah yep. sorry, i was testing the build locally so didn't revert back to latest.

Reply:
all good! 👍